### PR TITLE
BZ 2072784 updated text to match graphic

### PR DESCRIPTION
**Pull request by cowgirljessi**
Just clearing up any confusion between graphic and text.

Pursuing resolution of :https://bugzilla.redhat.com/show_bug.cgi?id=2072784

Netlify not showing changes, but here's the link:
https://docs.openshift.com/container-platform/4.6/networking/ingress-operator.html#nw-ingress-controller-endpoint-publishing-strategies_configuring-ingress

### CODE REVIEW

**Comment by cowgirljessi** [comment]
@zhaozhanqi QE check plz

Needs to be applied from 4.6 onward

File: modules/nw-ingress-controller-endpoint-publishing-strategies.adoc [hunk 1]
```diff
@@ -17,7 +17,7 @@ image::202_OpenShift_Ingress_0222_node_port.png[{product-title} Ingress NodePort
 The preceding graphic shows the following concepts pertaining to {product-title} Ingress NodePort endpoint publishing strategy:
 
 * All the available nodes in the cluster have their own, externally accessible IP addresses. The service running in the cluster is bound to the unique NodePort for all the nodes.
-* When the client connects to a node that is down, for example, by way of the `193.10.0.10` IP address in the graphic, the node port directly connects the client to an available node that is running the service. In this scenario, no load balancing is required. As the image shows, `193.10.0.10` is down and another IP address must be used instead.
+* When the client connects to a node that is down, for example, by way of the `10.0.128.4` IP address in the graphic, the node port directly connects the client to an available node that is running the service. In this scenario, no load balancing is required. As the image shows, `10.0.128.4` is down and another IP address must be used instead.
```
Comment (jldohmann) [inline modules/nw-ingress-controller-endpoint-publishing-strategies.adoc:20]: Is "by way of" an idiom? The ISG mentions to try and avoid them, I think they can be difficult to translate. Maybe "by connecting to" would be clearer in this situation, but I leave the decision up to you 😄 

Additionally, just a small suggestion to add an [object reference](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#object-references) for the second mention of the IP address. So in total I think the change would look something like this 🙂 :
```suggestion
* When the client connects to a node that is down, for example, by connecting to the `10.0.128.4` IP address in the graphic, the node port directly connects the client to an available node that is running the service. In this scenario, no load balancing is required. As the image shows, the `10.0.128.4` address is down and another IP address must be used instead.
```

**Review by jldohmann — COMMENTED** [review]
Minor nits, but LGTM!

**Comment by ahardin-rh** [comment]
Thank you!

**Pull request merged**